Zoo/src/filesInJava/Geography.java

```diff
@@ -0,0 +1,59 @@
+package filesInJava;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Created by admin-batch-a on 7/27/17.
+ */
+public class Geography {
+
+    public void writeToFile() throws IOException {
+        String filePath = "/home/admin-batch-a/Desktop/TestDevelopment/Projects/modern.core.java.repo/Zoo";
```

Comment:
This code will give error. There is no file name of it. Kindly double-check mine, it has like "other_folders/employee.txt" which is missing in your case.